apisix/plugins/kafka-logger.lua

```diff
@@ -128,6 +128,7 @@ local schema = {
         producer_max_buffering = {type = "integer", minimum = 1, default = 50000},
         producer_time_linger = {type = "integer", minimum = 1, default = 1},
         meta_refresh_interval = {type = "integer", minimum = 1, default = 30},
+        max_pending_entries = {type = "integer", minimum = 1, default = 2000}
```

Comment:
we can not set the default value to `2000`

the default way should be same as before

Reply:
no default value or `0` is acceptable